diffusion-model-with-mnist-data.py

```diff
@@ -167,3 +167,53 @@ def forward(self, noisy_images, timestep):
 # # Evaluation
 
 
+# -
+
+# # Sampling
+
+# +
+def sample_image(image_size, beta, noise_predictor, simple_variance=False):
```

Comment:
Replace image_size with x_t/noisy image, and get image_size by using image_size = x_t.shape / image_size = image.shape. This is important to reconstruct test images after the forward noising process.